RetailCoder.VBE/Navigation/CodeExplorer/CodeExplorerViewModel.cs

```diff
@@ -80,7 +80,7 @@ private void ParserState_StateChanged(object sender, EventArgs e)
         {
             Debug.WriteLine("CodeExplorerViewModel handles StateChanged...");
             IsBusy = _state.Status == ParserState.Parsing;
-            if (_state.Status != ParserState.Parsed)
+            if (_state.Status != ParserState.Ready)
```

Comment:
waiting for `Ready` state here would prevent the code explorer from showing *anything* in case of a parse error...